### PR TITLE
NUP-2299 Install pycapnp via nupic's extras to avoid embedding pycapnp's version info in .travis.yml.

### PR DESCRIPTION
**Pull request by vitaly-krugl**

### CODE REVIEW

**Comment by rhyolight** [comment]
👍 Give it a shot!

File: .travis.yml [hunk 1]
```diff
@@ -72,13 +72,13 @@ install:
...
-  - pip install pycapnp==0.5.8 --user
-
+  # NOTE nupic.bindings on *nix systems requires pycapnp, but it's not in
+  # nupic.bindings' requirements.txt yet for technical reasons, so rely on
+  # nupic's capnp extras to install the desired pycapnp.
   - NUPIC_VERSION=`cat ${NUPIC}/VERSION`
   - echo "Installing NuPIC ${NUPIC_VERSION} from wheelhouse..."
   - ls -l wheelhouse
-  - pip install --use-wheel --find-links=wheelhouse nupic==${NUPIC_VERSION} --user
+  - pip install --use-wheel --find-links=wheelhouse nupic[capnp]==${NUPIC_VERSION} --user
```
Comment (vitaly-krugl) [inline .travis.yml]: @oxtopus, I like this better. Also, in projects where we install nupic from local wheel, would something like the following work, or is something else needed? I didn't find an example for it in the pip docs.

```
pip install --user --ignore-installed -f wheelhouse/ dist/nupic-`cat VERSION`*.whl[capnp]
```

**Comment by vitaly-krugl** [comment]
@rhyolight, I am confused: why did fetch_wheels work in my other PR's build https://travis-ci.org/numenta/nupic.regression/builds/151900885 but not in this PR's build https://travis-ci.org/numenta/nupic.regression/builds/151951220?

**Comment by rhyolight** [comment]
@vitaly-krugl It's because the first PR was not run from a fork: 

> screen shot 2016-08-12 at 4 53 35 pm

The 2nd PR was run from `vitaly-krugl:nup2299-redo-pycapnp-install`. 

> screen shot 2016-08-12 at 4 53 44 pm

See https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables:

> Encryption and decryption keys are tied to the repository. If you fork a project and add it to Travis CI, it will have different keys than the original.

**Pull request merged**